app/views/dev_mode.py

```diff
@@ -54,11 +54,36 @@ def dev_mode():
                                  region_code=region_code,
                                  language_code=language_code,
                                  variant_flags=variant_flags)
+
         return redirect("/session?token=" + generate_token(payload).decode())
 
     return render_template("dev-page.html", user=os.getenv('USER', 'UNKNOWN'), available_schemas=available_schemas())
 
 
+@dev_mode_blueprint.route('/dev_flush', methods=['GET', 'POST'])
+def dev_flush_mode():
+    if request.method == "POST":
+        form = request.form
+        schema = form.get("schema")
+        eq_id, form_type = extract_eq_id_and_form_type(schema)
+        collection_exercise_sid = form.get("collection_exercise_sid")
+        ru_ref = form.get("ru_ref")
+
+        payload = {
```

Comment:
As above, the create_payload is not realistic as the token would never have the extra variables.